### PR TITLE
wifi-subsys: fix litle bugs

### PR DESCRIPTION
**Pull request by xkevin190**
### Contribution description

Some bugs were found in Wi-Fi at the time of the tests, these were:

1) error when trying to change wifi data once it is turned off
2) run wifi_stop in a row
3) switch from ap to ap_sta mode

### Testing procedure

to compile:
`idf.py build`

to flash and enabled console:
`idf.py -p /dev/tty.usbserial-0001 flash  monitor`

### CODE REVIEW

File: wifi-subsys/components/uart/src/at_handler.c [hunk 1]
```diff
@@ -183,6 +183,7 @@ void write_request(enum at_keys_n at_key, at_request_t *payload) {
         }
     }
 
+    printf("executed!!! here ");
```
Comment (xkevin190) [inline wifi-subsys/components/uart/src/at_handler.c:186]: remove

File: wifi-subsys/components/uart/src/subsys_uart.c [hunk 2]
```diff
@@ -101,7 +101,7 @@ void rx_receive(void *arg) {
                 if (data[0] == 0x41 && data[1] == 0x54) {
                     at_handler(data);
                 } else {
-                    sendData(__func__, "\r\n400\r\n");
+                    // sendData(__func__, "\r\n400\r\n");
```
Comment (luisan00) [inline wifi-subsys/components/uart/src/subsys_uart.c:104]:
```suggestion
```

**Review by luisan00 — CHANGES_REQUESTED** [review]
round one

**Review by luisan00 — APPROVED** [review]
:+1:  let's go

**Comment by xkevin190** [comment]
doing squash!!!

**Comment by luisan00** [comment]
almost ready to merge!

**Pull request merged**